FhirToDataLake/src/Microsoft.Health.Fhir.Synapse.HealthCheck/Checkers/AzureContainerRegistryHealthChecker.cs

```diff
@@ -19,7 +19,7 @@ namespace Microsoft.Health.Fhir.Synapse.HealthCheck.Checkers
 {
     public class AzureContainerRegistryHealthChecker : BaseHealthChecker
     {
-        private const string MediatypeV2Manifest = "application/vnd.docker.distribution.manifest.v2+json";
+        private const string ManifestMediatype = "application/vnd.oci.image.manifest.v1+json,application/vnd.docker.distribution.manifest.v2+json";
```

Comment:
Also need to update it in Microsoft.Health.Fhir.Synapse.Core/Jobs/ExternalDependencyChecker.cs

Reply:
Nice catch. Will fix it